eZ/Publish/Core/MVC/Symfony/Cache/Http/LocationAwareStore.php

```diff
@@ -1,255 +0,0 @@
-<?php
```

Comment:
> Hmm, we can add deprecation warnings, can't we ?

we can, so you prefer that then, and that deprecation warning says class most likely won't work anymore.